ci/update-versions.py

```diff
@@ -15,14 +15,18 @@
 }
 
 basedir = os.path.normpath(os.path.join(os.path.dirname(sys.argv[0]), '..'))
+github_token = os.getenv('GITHUB_TOKEN')
 
 with open(os.path.join(basedir, 'antora.yml'), 'r+') as fh:
     config = yaml.safe_load(fh)
     attrs = config.setdefault('asciidoc', {}).setdefault('attributes', {})
     orig_attrs = attrs.copy()
 
     for attr, repo in GITHUB_RELEASES.items():
-        resp = requests.get(f'https://api.github.com/repos/{repo}/releases/latest')
+        resp = requests.get(
+            f'https://api.github.com/repos/{repo}/releases/latest',
+            headers={'Authorization': f'Bearer {github_token}'} if github_token else {}
+        )
```

Comment:
optional: I found this a bit hard to read.. maybe making the assighment with the ternary operator in it up from would help?

```suggestion
        headers={'Authorization': f'Bearer {github_token}'} if github_token else {}
        resp = requests.get(
            f'https://api.github.com/repos/{repo}/releases/latest', headers=headers
        )
```